### ISSUE TITLE
[Feature Request] Responsivness

### ISSUE THREAD

**Issue by qoomon** [kyechan99/capsule-render#10]
Would be nice to have an option to prevent scaling text and keep height of resulting svg image e.g. https://raw.githubusercontent.com/matfantinel/matfantinel/master/waves.svg

**Comment by kyechan99** [comment]
Do you mean that you want to use it like adjusting the height with the `<img  height=' '/>` instead of params `/api/height=   ` ?

To sum up, do you want the height of the svg to follow the parent tag?

**Comment by qoomon** [comment]
exactly. Keep size and ratio of text, while scaling background to width of parent. (If possible, scaling down text, if too big to fit into parent tag)

**Comment by qoomon** [comment]
I think I found a solution see following example https://florian.ec/blog/responsive-svg-constant-font-size/example4-fluid.html and the corresponding toturialhttps://florian.ec/blog/responsive-svg-constant-font-size/

**Comment by tobealive** [comment]
+1 from me. It would be great to have a static font size option.

The scaling difference on mobile and desktop is drastic atm.
